pkg/kapis/devops/v1alpha4/handler.go

```diff
@@ -71,3 +74,40 @@ func (h *apiHandler) listPipelineRuns() restful.RouteFunction {
 		_ = response.WriteAsJson(apiResult)
 	}
 }
+
+func (h *apiHandler) createPipelineRuns() restful.RouteFunction {
+	return func(request *restful.Request, response *restful.Response) {
```

Comment:
Why don't just add parameters `request *restful.Request, response *restful.Response` into function `createPipelineRuns` directly. See [this example](https://github.com/kubesphere/ks-devops/blob/4a8b4609f40de66f448607212333ff4dfbdaadab/pkg/kapis/devops/v1alpha2/devops.go#L909).